code/mathematical-algorithms/factorial/logfactorial.cpp

```diff
@@ -0,0 +1,20 @@
+#include <iostream>
```

Comment:
In this code, you are calculating the log of a factorial but the category is devoted to the calculation of factorial. Kindly include this in a new folder `log_of_factorial` under `mathematical_algorithms/basic` 👍